subsys/bluetooth/host/conn.c

```diff
@@ -2319,19 +2319,6 @@ struct bt_conn *bt_conn_create_slave_le(const bt_addr_le_t *peer,
 	if (conn) {
 		switch (conn->state) {
 		case BT_CONN_CONNECT_DIR_ADV:
-			/* Handle the case when advertising is stopped with
-			 * bt_le_adv_stop function
-			 */
-			err = bt_le_adv_start_internal(&param_int, NULL, 0,
-						       NULL, 0, peer);
-			if (err && (err != -EALREADY)) {
-				BT_WARN("Directed advertising could not be"
-					" started: %d", err);
-				bt_conn_unref(conn);
-				return NULL;
-			}
-
-		case BT_CONN_CONNECT:
```

Comment:
Why is the `BT_CONN_CONNECT` case removed?

Reply:
The connect state should be for initiator, after calling le_create_conn command. Shouldn't be a valid state for an advertiser.

Reply:
Ok, this wasn't clear from the commit message.

Reply:
Good point. I forgot to mention it.